README.md

```diff
@@ -3,3 +3,10 @@
 🌱 I’m currently learning Scripting
 💞️ I’m looking to collaborate on everyone one that have the same interest in Cybersecurity.
 📫 How to reach me with my email: nguonsg@mail.uc.edu
+Make sure you completed the following items before you submit the assignment on Canvas:
+
+ Create a project on GitHub
+ Create a branch on GitHub
+ Make changes to the branch
+ Open a pull request on GitHub
+ Merge your pull request
```

Comment:
```suggestion
* Create a project on GitHub
* Create a branch on GitHub
* Make changes to the branch
* Open a pull request on GitHub
* Merge your pull request
```
This will created a bulleted list